README.md

```diff
@@ -1,10 +1,10 @@
-## Amazon Ion Hive Serde
+## Amazon Ion Hive2 Serde
 
-A Apache Hive SerDe (short for serializer/deserializer) for the Ion file format.
+An Apache Hive SerDe (short for serializer/deserializer) for the Ion file format, this package supports Hive 2.
 
-[![Build Status](https://travis-ci.org/amzn/ion-hive-serde.svg?branch=master)](https://travis-ci.org/amzn/ion-hive-serde)
+[![Build Status](https://github.com/amzn/ion-hive-serde/actions/workflows/main.yml/badge.svg?branch=hive2)](https://github.com/amzn/ion-hive-serde/actions?query=branch%3Ahive2+)
 [![Maven Central](https://maven-badges.herokuapp.com/maven-central/com.amazon.ion/ion-hive-serde/badge.svg)](https://maven-badges.herokuapp.com/maven-central/com.amazon.ion/ion-hive-serde)
```

Comment:
The artifact id should change here too, yes?

Reply:
Yes, 

I will update maven center and javadoc's links once we distribute artifacts to Maven, just in case I update a wrong link before the artifacts exist in Maven.